src/containers/Projects/CreateProjectPopup/index.js

```diff
@@ -104,21 +102,16 @@ const onSubmit = async (values, dispatch, props) => {
       }
     }
   }
-  };
+};
 
 export const uploadThumb = async (e, props) => {
   const formData = new FormData();
   try {
     formData.append('thumb', e.target.files[0]);
-
     imageValidation = '';
     await props.uploadProjectThumbnail(formData);
   } catch (err) {
-    Swal.fire({
-      icon: 'error',
-      title: 'Error',
-      text: 'Failed to upload thumb.',
-    });
+    console.log(err);
```

Comment:
Same here

Reply:
it is being taken care from  actions/services